### PR TITLE
Preprod: movie: Cosmic Sin (2021) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
More like cosmic slop, and I don't mean the 1994 anthology movie "Cosmic Slop", which was actually pretty good. Rating 1/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 1]
```diff
@@ -791,6 +791,49 @@ murder [Zach was exposed for having deliberately electrocuted Richard Alling to
...
+:: References
+https://en.wikipedia.org/wiki/Cosmic_Sin
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+attack from outer space by a powerful conquering enemy [It was up to Ford and his team of elite soldiers to stop the malevolent Sigea aliens from conquering Earth.]
+extraterrestrial being [The malevolent Sigea aliens had designed on taking over the Earth.]
+weapons of mass destruction [After much build up, a "Q-bomb" super-weapon used to destroy the Sigea's attack armada and home star system, saving Earth from being conquered.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:820]: Sentence does not parse...

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 2]
```diff
@@ -791,6 +791,49 @@ murder [Zach was exposed for having deliberately electrocuted Richard Alling to
...
+
+:: References
+https://en.wikipedia.org/wiki/Cosmic_Sin
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+attack from outer space by a powerful conquering enemy [It was up to Ford and his team of elite soldiers to stop the malevolent Sigea aliens from conquering Earth.]
+extraterrestrial being [The malevolent Sigea aliens had designed on taking over the Earth.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:819]: I don't think you can have "designs on [verb phrase]". But you can have designs on Sweden, or the throne, for example.

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 3]
```diff
@@ -791,6 +791,49 @@ murder [Zach was exposed for having deliberately electrocuted Richard Alling to
...
+
+:: Minor Themes
+colonization of Mars [The viewer learns that the first colony on Mars was founded in the year 2031, but had somehow catastrophically failed in 2281.]
+space colonization [The Alliance had a colony, named Ellora, on some far-flung planet.]
+mechanical man [A humanoid robot bartender struggled to pour Ford a shot of what appeared to be whisky.]
+atrocities of war [The viewer learns that Ford, under Alliance orders, dropped a "Q-bomb" on a rebel colony, killing 70 million people.]
+duty to disobey illegal orders [Ford stated that he was just following Alliance orders in response to being called out for using a weapon of mass destruction, which resulted in the deaths of 70 million people.]
+space portal [Ford and his party traveled to Ellora via a "Quantum Jump Gate" to stop the Sigea aliens.]
+hologram [A holographic country music singer was playing on stage at the bar.] {49:00, 1:24:00}
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:830]: Don't really care, but I thought you had decided not to leave {}'s?

**Review by odinlake — COMMENTED** [review]
Typos noted

File: notes/stories/film/film-scifi-2020s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt]: Rephrased that one.

File: notes/stories/film/film-scifi-2020s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt]: Rephrased.

File: notes/stories/film/film-scifi-2020s.st.txt (hunk 3 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt]: Whoops. I missed taking those {} out. They've now been eliminated.

**Pull request merged**